### PR TITLE
Added preview feature for uploaded images

### LINKED ISSUE #164: Images preview not available in the "edit project" form
For example, in the form below, next to the image field, it says 2 images added but there is no preview of those images available: 
Screen Shot 2021-05-25 at 5 39 17 PM

Ideally, there are image previews shown in the edit project form.

### PR DESCRIPTION
**Pull request by iamhks**
## Summary

Description of PR here...
Closes #164 

Screenshots:
Screenshot 2023-03-21 at 01 09 49
Screenshot 2023-03-21 at 01 09 29

### CODE REVIEW

File: zubhub_frontend/zubhub/public/index.html [hunk 1]
```diff
@@ -47,6 +47,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Zubhub</title>
+    <script src="https://kit.fontawesome.com/58aa8d2b83.js" crossorigin="anonymous"></script>
```
Comment (tuxology) [inline zubhub_frontend/zubhub/public/index.html:50]: Can we avoid using this and reuse parts that we already are using? Check imagePreview feature out that was done by @souad988 for the activity feature :) https://github.com/unstructuredstudio/zubhub/blob/master/zubhub_frontend/zubhub/src/assets/js/styles/views/create_activity/createActivityStyles.js#L51-L69

**Comment by tuxology** [comment]
Thanks for this @iamhks 👍 Can we reuse some parts from the image preview work done already by @souad988  here? https://github.com/unstructuredstudio/zubhub/blob/master/zubhub_frontend/zubhub/src/components/upload_file/uploadFileScripts.js#L96-L116

File: zubhub_frontend/zubhub/public/index.html (hunk 1 above)
Comment (tuxology) [inline zubhub_frontend/zubhub/public/index.html]: It would be nice if we can reuse some earlier and work and avoid adding new FA js

Comment (iamhks) [reply zubhub_frontend/zubhub/public/index.html]: Updated the PR

**Comment by iamhks** [comment]
@tuxology I have updated the PR to show images just like create activity screen.

Although what I said [here](https://github.com/unstructuredstudio/zubhub/issues/164#issuecomment-1476843811) is still valid and as expected the pictures show up just like expected:
Screenshot 2023-03-25 at 03 54 44

But I also think it is nice to have the same consistent design throughout the site so preview Image in createProject replicates createActivity's 2nd screen. 
I have also updated the caption to show dynamic number of images as per addition/removal like here:
Screenshot 2023-03-25 at 13 12 18

Pls review at your convenience.

**Comment by tuxology** [comment]
@iamhks Thanks, this is shaping up nice! I'll be testing this, but in the meantime one final nimor thing: can we use the close buttons same as what @psalmuelle used in his PR https://github.com/unstructuredstudio/zubhub/pull/613 ? This will allow us to have consistent closing buttons. This way we have same closing buttons everywhere. We can also use what you have but just ensure the background is white and not transparent for the close buttons. 

Please also test this for responsiveness and share screenshots if possible 🙏

**Comment by iamhks** [comment]
Sure @tuxology, I have made the changes and here are the screenshots:

Screenshot 2023-03-26 at 09 13 55

Screenshot 2023-03-26 at 09 12 54

Screenshot 2023-03-26 at 09 14 39

Screenshot 2023-03-26 at 09 15 09

**Comment by iamhks** [comment]
@tuxology Are we good with this PR to merge?

**Comment by iamhks** [comment]
Hi @tuxology Any update on this?

**Comment by tuxology** [comment]
Done as part of #718 Closing this.

**Pull request closed without merging**